roles/openshift_logging_eventrouter/templates/eventrouter-dc.j2

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 kind: DeploymentConfig
 metadata:
   name: logging-eventrouter
-  namespace: logging
+  namespace: "{{ namespace }}"
```

Comment:
I'm not certain we need to specify this here if oc_obj correctly specifies the namespace upon creation.  The namespace will be set by the action of creating the DC